src/main/java/org/owasp/esapi/filters/SecurityWrapperRequest.java

```diff
@@ -681,15 +681,15 @@ public String getServerName() {
      * HttpServletRequest after parsing and checking the range 0-65536.
      * @return The local server port
      */
-	public int getServerPort() {
-		int port = getHttpServletRequest().getServerPort();
-		if ( port < 0 || port > 0xFFFF ) {
-			logger.warning( Logger.SECURITY_FAILURE, "HTTP server port out of range: " + port );
-			port = 0;
-		}
-		return port;
-	}
- 	
+    public int getServerPort() {
```

Comment:
I hate to ask this but with the tab to space transformation in this file alone, might it be worthwhile once we agree we're at the "end" of this release that we apply it to the whole project and then create a single commit with one change that is ONLY whitespace?

Reply:
You sure you referenced the correct file here? I just checked and I see no tabs at all in src/main/java/org/owasp/esapi/filters/SecurityWrapperRequest.java.

Reply:
@xeno6696 - BTW I usually fix these by running ':retab' in gvim which converts all the tabs to the appropriate # of spaces with the tabsize set to 4 spaces.